docs/tutorials/contributing/strategy/writing_the_new_strategy.rst

```diff
@@ -7,7 +7,7 @@ Identify a new strategy
 The library has a large number of strategies::
 
     >>> import axelrod as axl
-    >>> len(axl.strategies)
+    >>> len(axl.strategies) # doctest: +SKIP
     103
 
 If you're not sure if you have a strategy that has already been implemented
```

Comment:
I think that if we have the length of strategies in the docs then it should be tested (to make sure it's accurate). If we're going to ignore it then we should just remove it.

I would rather it was left in but yeah, not sure how to get around it.